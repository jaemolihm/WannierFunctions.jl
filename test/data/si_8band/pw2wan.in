&inputpp
   outdir = 'temp/'
   prefix = 'si'
   seedname = 'si'
   write_mmn = .true.
   write_amn = .true.
   write_dmn = .true.
/
