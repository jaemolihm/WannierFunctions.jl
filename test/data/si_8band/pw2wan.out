MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found

     Program PW2WANNIER v.7.0 starts on 30Jul2022 at 16:17: 7 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI & OpenMP), running on       4 processor cores
     Number of MPI processes:                 4
     Threads/MPI process:                     1

     MPI processes distributed on     1 nodes
     R & G space division:  proc/nbgrp/npool/nimage =       4
     832 MiB available memory on the printing compute node when the environment starts
 
 
  Reading nscf_save data

     Reading xml data from directory:

     temp/si.save/

     IMPORTANT: XC functional enforced from input :
     Exchange-correlation= PW
                           (   1   4   0   0   0   0   0)
     Any further DFT definition will be discarded
     Please, verify this is what you really want

 
     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min         256     256     77                 5385     5385     881
     Max         257     257     79                 5388     5388     882
     Sum        1027    1027    313                21547    21547    3527
 
     Using Slab Decomposition
 
     Reading collected, re-writing distributed wavefunctions
 
  Spin CASE ( default = unpolarized )
 
  Wannier mode is: standalone     
 
  -----------------
  *** Reading nnkp 
  -----------------
 
  Checking info from wannier.nnkp file
 
  - Real lattice is ok
  - Reciprocal lattice is ok
  - K-points are ok
  - Number of wannier functions is ok (  8)
  - All guiding functions are given 
 
 Projections:
   -0.125000    0.125000    0.125000 -3  1  1    1.000000
   -0.125000    0.125000    0.125000 -3  2  1    1.000000
   -0.125000    0.125000    0.125000 -3  3  1    1.000000
   -0.125000    0.125000    0.125000 -3  4  1    1.000000
    0.125000   -0.125000   -0.125000 -3  1  1    1.000000
    0.125000   -0.125000   -0.125000 -3  2  1    1.000000
    0.125000   -0.125000   -0.125000 -3  3  1    1.000000
    0.125000   -0.125000   -0.125000 -3  4  1    1.000000
 
  Reading data about k-point neighbours 
 
  All neighbours are found 
 
  Opening pp-files 
 
 
  ----------------
  *** Compute DMN 
  ----------------
 
  Number of symmetry operators =    48
      1-th symmetry operators is
      1.0000000      0.0000000      0.0000000
      0.0000000      1.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
      0.0000000      0.0000000      0.0000000
      2-th symmetry operators is
     -1.0000000      0.0000000      0.0000000
      0.0000000     -1.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
     -0.2500000      0.2500000      0.0000000
      3-th symmetry operators is
     -1.0000000      0.0000000      0.0000000
      0.0000000      1.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
     -0.2500000      0.0000000      0.2500000
      4-th symmetry operators is
      1.0000000      0.0000000      0.0000000
      0.0000000     -1.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
      0.0000000      0.2500000      0.2500000
      5-th symmetry operators is
      0.0000000      1.0000000      0.0000000
      1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
      0.0000000      0.0000000      0.0000000
      6-th symmetry operators is
      0.0000000     -1.0000000      0.0000000
     -1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
     -0.2500000      0.2500000      0.0000000
      7-th symmetry operators is
      0.0000000      1.0000000      0.0000000
     -1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
      0.0000000      0.2500000      0.2500000
      8-th symmetry operators is
      0.0000000     -1.0000000      0.0000000
      1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
     -0.2500000      0.0000000      0.2500000
      9-th symmetry operators is
      0.0000000      0.0000000      1.0000000
      0.0000000     -1.0000000      0.0000000
      1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000      0.0000000
     10-th symmetry operators is
      0.0000000      0.0000000     -1.0000000
      0.0000000     -1.0000000      0.0000000
     -1.0000000      0.0000000      0.0000000
     -0.2500000      0.0000000      0.2500000
     11-th symmetry operators is
      0.0000000      0.0000000      1.0000000
      0.0000000      1.0000000      0.0000000
     -1.0000000      0.0000000      0.0000000
      0.0000000      0.2500000      0.2500000
     12-th symmetry operators is
      0.0000000      0.0000000     -1.0000000
      0.0000000      1.0000000      0.0000000
      1.0000000      0.0000000      0.0000000
     -0.2500000      0.2500000      0.0000000
     13-th symmetry operators is
     -1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
      0.0000000      1.0000000      0.0000000
      0.0000000      0.2500000      0.2500000
     14-th symmetry operators is
     -1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
      0.0000000     -1.0000000      0.0000000
      0.0000000      0.0000000      0.0000000
     15-th symmetry operators is
      1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
      0.0000000     -1.0000000      0.0000000
     -0.2500000      0.2500000      0.0000000
     16-th symmetry operators is
      1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
      0.0000000      1.0000000      0.0000000
     -0.2500000      0.0000000      0.2500000
     17-th symmetry operators is
      0.0000000      1.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
      1.0000000      0.0000000      0.0000000
     -0.2500000      0.0000000      0.2500000
     18-th symmetry operators is
      0.0000000     -1.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
     -1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000      0.0000000
     19-th symmetry operators is
      0.0000000      1.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
     -1.0000000      0.0000000      0.0000000
     -0.2500000      0.2500000      0.0000000
     20-th symmetry operators is
      0.0000000     -1.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
      1.0000000      0.0000000      0.0000000
      0.0000000      0.2500000      0.2500000
     21-th symmetry operators is
      0.0000000      0.0000000      1.0000000
      1.0000000      0.0000000      0.0000000
      0.0000000      1.0000000      0.0000000
     -0.2500000      0.2500000      0.0000000
     22-th symmetry operators is
      0.0000000      0.0000000      1.0000000
     -1.0000000      0.0000000      0.0000000
      0.0000000     -1.0000000      0.0000000
     -0.2500000      0.0000000      0.2500000
     23-th symmetry operators is
      0.0000000      0.0000000     -1.0000000
     -1.0000000      0.0000000      0.0000000
      0.0000000      1.0000000      0.0000000
      0.0000000      0.0000000      0.0000000
     24-th symmetry operators is
      0.0000000      0.0000000     -1.0000000
      1.0000000      0.0000000      0.0000000
      0.0000000     -1.0000000      0.0000000
      0.0000000      0.2500000      0.2500000
     25-th symmetry operators is
     -1.0000000      0.0000000      0.0000000
      0.0000000     -1.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
      0.0000000      0.0000000      0.0000000
     26-th symmetry operators is
      1.0000000      0.0000000      0.0000000
      0.0000000      1.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
     -0.2500000      0.2500000      0.0000000
     27-th symmetry operators is
      1.0000000      0.0000000      0.0000000
      0.0000000     -1.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
     -0.2500000      0.0000000      0.2500000
     28-th symmetry operators is
     -1.0000000      0.0000000      0.0000000
      0.0000000      1.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
      0.0000000      0.2500000      0.2500000
     29-th symmetry operators is
      0.0000000     -1.0000000      0.0000000
     -1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
      0.0000000      0.0000000      0.0000000
     30-th symmetry operators is
      0.0000000      1.0000000      0.0000000
      1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
     -0.2500000      0.2500000      0.0000000
     31-th symmetry operators is
      0.0000000     -1.0000000      0.0000000
      1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
      0.0000000      0.2500000      0.2500000
     32-th symmetry operators is
      0.0000000      1.0000000      0.0000000
     -1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
     -0.2500000      0.0000000      0.2500000
     33-th symmetry operators is
      0.0000000      0.0000000     -1.0000000
      0.0000000      1.0000000      0.0000000
     -1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000      0.0000000
     34-th symmetry operators is
      0.0000000      0.0000000      1.0000000
      0.0000000      1.0000000      0.0000000
      1.0000000      0.0000000      0.0000000
     -0.2500000      0.0000000      0.2500000
     35-th symmetry operators is
      0.0000000      0.0000000     -1.0000000
      0.0000000     -1.0000000      0.0000000
      1.0000000      0.0000000      0.0000000
      0.0000000      0.2500000      0.2500000
     36-th symmetry operators is
      0.0000000      0.0000000      1.0000000
      0.0000000     -1.0000000      0.0000000
     -1.0000000      0.0000000      0.0000000
     -0.2500000      0.2500000      0.0000000
     37-th symmetry operators is
      1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
      0.0000000     -1.0000000      0.0000000
      0.0000000      0.2500000      0.2500000
     38-th symmetry operators is
      1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
      0.0000000      1.0000000      0.0000000
      0.0000000      0.0000000      0.0000000
     39-th symmetry operators is
     -1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
      0.0000000      1.0000000      0.0000000
     -0.2500000      0.2500000      0.0000000
     40-th symmetry operators is
     -1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
      0.0000000     -1.0000000      0.0000000
     -0.2500000      0.0000000      0.2500000
     41-th symmetry operators is
      0.0000000     -1.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
     -1.0000000      0.0000000      0.0000000
     -0.2500000      0.0000000      0.2500000
     42-th symmetry operators is
      0.0000000      1.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
      1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000      0.0000000
     43-th symmetry operators is
      0.0000000     -1.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
      1.0000000      0.0000000      0.0000000
     -0.2500000      0.2500000      0.0000000
     44-th symmetry operators is
      0.0000000      1.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
     -1.0000000      0.0000000      0.0000000
      0.0000000      0.2500000      0.2500000
     45-th symmetry operators is
      0.0000000      0.0000000     -1.0000000
     -1.0000000      0.0000000      0.0000000
      0.0000000     -1.0000000      0.0000000
     -0.2500000      0.2500000      0.0000000
     46-th symmetry operators is
      0.0000000      0.0000000     -1.0000000
      1.0000000      0.0000000      0.0000000
      0.0000000      1.0000000      0.0000000
     -0.2500000      0.0000000      0.2500000
     47-th symmetry operators is
      0.0000000      0.0000000      1.0000000
      1.0000000      0.0000000      0.0000000
      0.0000000     -1.0000000      0.0000000
      0.0000000      0.0000000      0.0000000
     48-th symmetry operators is
      0.0000000      0.0000000      1.0000000
     -1.0000000      0.0000000      0.0000000
      0.0000000      1.0000000      0.0000000
      0.0000000      0.2500000      0.2500000


  DMN(d_matrix_wann): nir =        4
       1       2       3       4
  DMN(d_matrix_wann) calculated


  DMN(d_matrix_band): nir =        4
  DMN(d_matrix_band): nir in this pool =        4
       1       2       3       4
  DMN(d_matrix_band) calculated
 
  --------------------------
  *** Compute  A projections
  --------------------------
 
 Number of local k points =       27
       1       2       3       4       5       6       7       8       9      10
      11      12      13      14      15      16      17      18      19      20
      21      22      23      24      25      26      27
  AMN calculated
 
  ---------------
  *** Compute  M 
  ---------------
 
 Number of local k points =       27
       1       2       3       4       5       6       7       8       9      10
      11      12      13      14      15      16      17      18      19      20
      21      22      23      24      25      26      27
  MMN calculated
 
  -----------------------------------
  *** Orbital terms are not computed 
  -----------------------------------
 
  -----------------------------------
  *** SHC terms are not computed 
  -----------------------------------
 
  ----------------
  *** Write bands 
  ----------------
 
 
  -----------------------------
  *** Plot info is not printed 
  -----------------------------
 
  -----------------------------
  *** Parity info is not printed 
  -----------------------------
 
  ------------
  *** Stop pp 
  ------------
 
 
     init_pw2wan  :      0.17s CPU      0.60s WALL (       1 calls)
     compute_dmn  :      2.18s CPU      2.24s WALL (       1 calls)
     compute_amn  :      2.28s CPU      2.29s WALL (       1 calls)
     compute_mmn  :      0.57s CPU      0.61s WALL (       1 calls)

     Internal routines:
     compute_u_kb :      0.51s CPU      0.53s WALL (     216 calls)
 
     PW2WANNIER   :      5.20s CPU      5.77s WALL

 
   This run was terminated on:  16:17:12  30Jul2022            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
 
